cv32e40p/sim/ExternalRepos.mk

```diff
@@ -13,9 +13,12 @@
 
 export SHELL = /bin/bash
 
-CV_CORE_REPO   ?= https://github.com/openhwgroup/cv32e40p
-CV_CORE_BRANCH ?= master
-CV_CORE_HASH   ?= d0d1c25
+#CV_CORE_REPO   ?= https://github.com/openhwgroup/cv32e40p
+#CV_CORE_BRANCH ?= dev
+#CV_CORE_HASH   ?= bddbd38fb0b5ced6208090691002b7ee021dcc8d
+CV_CORE_REPO   ?= https://github.com/YoannPruvost/cv32e40p
+CV_CORE_BRANCH ?= dev_rvfi_clean
```

Comment:
Not `dev_rvfi_2`?  I ask because [cv32e40p pull-request #743](https://github.com/openhwgroup/cv32e40p/pull/743) was for that branch.

Reply:
Yes
I am waiting for this PR to be merged so I can put the openHW repo and the correct branch and hash

Reply:
Understood.  I will hold off on the approval and merge until [cv32e40p pull-request #743](https://github.com/openhwgroup/cv32e40p/pull/743) is merged in.